tests/e2e/utils/requestHandlers/headers/CheMultiuserAuthorizationHeaderHandler.ts

```diff
@@ -12,13 +12,26 @@ import { IAuthorizationHeaderHandler } from './IAuthorizationHeaderHandler';
 import { inject, injectable } from 'inversify';
 import { DriverHelper } from '../../DriverHelper';
 import { CLASSES } from '../../../inversify.types';
+import { Logger } from '../../Logger';
 
 @injectable()
 export class CheMultiuserAuthorizationHeaderHandler implements IAuthorizationHeaderHandler {
+    static AUTHORIZATION_TOKEN: string;
```

Comment:
`AUTHORIZATION_TOKEN` is not actually a constant, but regular variable, which is obtaining the value in line 26 below.

So, no need to capitalize the name, to avoid confusion.

Reply:
hmm, good point, maybe it doesn't need to be

Reply:
thanks for considering